config/locales/en.yml

```diff
@@ -503,6 +503,7 @@ en:
       subtitle_html: Ask for new maintainers or request ownership <a class="adoption__blog__link" href="https://blog.rubygems.org/2022/01/19/rubygems-adoptions.html">(read more)</a>
     edit:
       change_avatar: Change Avatar
+      disabled_avatar_html: "Default avatar is used due to private email settings. To enable a personalized <a href='https://gravatar.com'>Gravatar</a>, turn on 'Show email in public profile'. Notice this will expose your email in public."
```

Comment:
```suggestion
      disabled_avatar_html: "A default avatar is used due to private email settings. To enable a personalized <a href='https://gravatar.com'>Gravatar</a>, turn on 'Show email in public profile'. Notice this will expose your email to the public."
```

Reply:
Thanks for suggestion, I did my best to generate most friendly message with GPT to be able to open PR with something at least for initial discussion. This looks much smoother.